script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -454,7 +454,7 @@ and those who came after are the young blood."
 -- Page 605, Offset 18092.
 -- 吸血鬼の尺度で言うと、ヴローヴはつい最近祖になった死徒である、と。
 From a vampire's perspective,
- Vlov was a Dead Apostle that had only recently become an Ancestor.
+ Vlov is a Dead Apostle that has only recently become an Ancestor.
```

Comment:
I'd keep past tense, he's dead at this point (I assume??)

Reply:
True, lost the context there for a second.